tests/functional_tests/CMakeLists.txt

```
@@ -78,4 +78,7 @@ if (REQUESTS_OUTPUT STREQUAL "OK")
 else()
   message(WARNING "functional_tests_rpc and check_missing_rpc_methods skipped, needs the 'requests', 'psutil', 'monotonic', 'zmq', and 'deepdiff' python modules")
   set(CTEST_CUSTOM_TESTS_IGNORE ${CTEST_CUSTOM_TESTS_IGNORE} functional_tests_rpc check_missing_rpc_methods)
+  if (EXPECT_FUNCTIONAL_TESTS)
+    message(FATAL_ERROR "Functional tests are expected")
```

```suggestion
    message(FATAL_ERROR "Functional tests are expected, but aforementioned python dependencies are not found")
```

Just in case they wouldn't understand why it doesn't work...